### PR TITLE
Implement Insufficient Funds Error

### LINKED ISSUE #3914: Expenditures: As a user I can see an error for insufficient funds to claim
[Provided by Adrian via Contractors project management platform]

## Description
User has to be able to see an error for insufficient funds to claim.

## Acceptance Criteria

- an option to see an error message above stages section
   - single funding source "Insufficient funds in team. Ensure team is funded."
   - multiple funding sources "Insufficient funds in team. Ensure all teams are funded."
- an option to see red dot, that should appear next to a funding source with insufficient funds
   - tooltip on hover (text "Required fields error")
- an option to see insufficient funds with red font

## Design

- [Figma Link - Insufficient funds](https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=5606%3A320150)
- [Figma Link - Insufficient funds - multiple](https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=5706%3A324464)

Single streams
32832211-8f467b82afb31ea44456c364ddf1dfd6

Multiple streams
32832222-a0d951d323979e56384ecbd6a19af340

### PR DESCRIPTION
**Pull request by joannasia**
## Description

- Streaming locked fundings - on claim funds click lighten up insufficient fields

## TODO

- Add logic to mock functionality.
- Waiting for [3845-active-stream](https://github.com/JoinColony/colonyDapp/tree/feature/3845-active-stream) to be merged to the phase-3, and went it;s approved and done, then merge this branch with target branch [feature/3846-locked-funding-source](https://github.com/JoinColony/colonyDapp/tree/feature/3846-locked-funding-source) - bc right now there are changes also from active-stream branch

Resolves #3914

### CODE REVIEW

**Comment by joannasia** [comment]
@willm30 HI, this one is still not ready for CR, it is blocked by other MR which needs to be merged firstly :)

**Review by arrenv — CHANGES_REQUESTED** [review]
It is looking good. With the mock data, it does not really allow you to test it properly though. I think it would be worthwhile adding in the actual balances. How to do this is shown here - src/modules/dashboard/components/ColonyHome/ColonyFunding/ColonyFunding.tsx

It would also help to then be able to ensure that it is identifying the right team that does not have enough funds. It seems to just mark the first one, which would seem a lot of logic is missing.

97a9603c-ab48-46c7-876b-492f6c8240ea

File: src/modules/dashboard/components/ExpenditurePage/Streaming/LockedStreaming.tsx [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
-import { CollapseExpandButtons } from '~dashboard/ExpenditurePage/Payments';
 import { Colony } from '~data/index';
 import Icon from '~core/Icon';
 
 import { Stage } from '../Stages/constants';
 
-import LockedFundingSource from './LockedFundingSource';
 import { FundingSource } from './types';
 import styles from './Streaming.css';
+import SingleLockedFunding from './SingleLockedFunding';
```
Comment (ArmandoGraterol) [inline src/modules/dashboard/components/ExpenditurePage/Streaming/LockedStreaming.tsx:12]: This needs to go above style imports, style imports are always last.

**Review by ArmandoGraterol — DISMISSED** [review]
Apart from the things Mr. Arren already pointed, I noticed some small things on the code

**Review by ArmandoGraterol — COMMENTED** [review]
For ETH, it's working as expected however for the native token of the colony it's still not detecting that there are not enough funds to pay:
FireShot Capture 755 - Create Expenditure - Colony - wonker - localhost
FireShot Capture 756 - Create Expenditure - Colony - wonker - localhost

**Comment by agnieszkajarosikloj** [comment]
@arrenv @ArmandoGraterol  I have added changes. Can you have a look?

**Review by arrenv — APPROVED** [review]
Looking to be working much better thank you @agnieszkajarosikloj for your hard work.

I seemed to notice one issue, where the root team which had enough funds was marked with a red dot, but the value was not highlighted red.

ca18dd0d-6b53-4bd6-9cb4-0c39bf34ae2e

**Review by chinins — APPROVED** [review]
All good other than what Arren noticed.

**Pull request merged**